proposal/color-4-new-spaces.md

```diff
@@ -0,0 +1,1864 @@
+in certain circumstances.
+
+* `hsl`:
+
+  * If the `saturation` value is `0%`, then the `hue` channel is powerless.
+
+  * If the `lightness` value is either `0%` or `100%`, then both the `hue` and
+    `saturation` values are powerless.
+
+* `hwb`:
+
+  * If the combined `whiteness` and `blackness` values (after normalization)
+    are equal to `100%`, then the `hue` channel is powerless.
+
+* `lab`/`oklab`:
+
+  * If the `lightness` value is `0%`, then both the `a` and `b` channels are
+    powerless.
+
+  > The current spec has an inline issue asking if high values of
+  > `lightness` (whites) should make the `a` and `b` values powerless:
+  > See: https://drafts.csswg.org/css-color-4/#issue-e05ac5c3
+
+* `lch`/`oklch`:
+
+  * If the `chroma` value is 0%, then the `hue` channel is powerless.
+
+  * If the `lightness` value is `0%`, then both the `hue` and `chroma` channels
+    are powerless.
+
+  > The current spec has an inline issue asking if high values of
+  > `lightness` (whites) should make the `hue` and `chroma` values powerless.
+  > See: https://drafts.csswg.org/css-color-4/#issue-1813c844
+
+### Color Interpolation Method
+
+A _color interpolation method_ is a space-separated list of unquoted strings,
+parsed according to the following syntax definition:
+
+<x><pre>
+**ColorInterpolationMethod** ::= 'in' (
+&#32;                                 RectangularColorSpace
+&#32;                               | PolarColorSpace HueInterpolationMethod?
+&#32;                             )
+**RectangularColorSpace**    ::= 'srgb'
+&#32;                          | 'srgb-linear'
+&#32;                          | 'lab'
+&#32;                          | 'oklab'
+&#32;                          | 'xyz'
+&#32;                          | 'xyz-d50'
+&#32;                          | 'xyz-d65'
+**PolarColorSpace**          ::= 'hsl'
+&#32;                          | 'hwb'
+&#32;                          | 'lch'
+&#32;                          | 'oklch'
+**HueInterpolationMethod**   ::= (
+&#32;                                'shorter'
+&#32;                              | 'longer'
+&#32;                              | 'increasing'
+&#32;                              | 'decreasing'
+&#32;                              | 'specified'
+&#32;                            ) 'hue'
+</pre></x>
+
+The resulting _interpolation color space_ is the [known color space] whose name
+is given by either the `PolarColorSpace` or `RectangularColorSpace` productions.
+
+> Different color interpolation methods provide different advantages. For that
+> reason, individual color procedures and functions can establish their own
+> color interpolation defaults, or provide a syntax for authors to explicitly
+> choose the method that best fits their need. The [CSS Color Level 4][color-4]
+> specification provides [additional guidance][default-space] for determining
+> appropriate defaults.
+
+[default-space]: https://drafts.csswg.org/css-color-4/#interpolation-space
```

Comment:
should this go to "https://www.w3.org/TR/css-color-4/#interpolation-space"?

Reply:
That's a good question. The published version is certainly more official. But for a spec that's still being worked on, it can also be more out of date, since an ED can have officially resolved changes just waiting to be published. @nex3 do you have a preference?

Reply:
Given that CSS Colors 4 is already at the CR state, I suggest linking to the published version. This way, the Sass spec won't need to be updated when it reaches the recommendation state. But I'll let @nex3 give her opinion there.
Implementors might still look at the ED for existing clarifications in the meantime.

According to the published version, it is expected to stay at CR state until at least October 5 to gather feedback. As Colors 4 is in the scope of Interop2022, we might expect it to move forward after that date.

Reply:
```suggestion
[default-space]: https://www.w3.org/TR/css-color-4/#interpolation-space
```

Reply:
I don't have a strong opinion here, but if the published version has all the terms we explicitly reference I guess it's probably the better option.